pkg/idtools/idtools_windows.go

```diff
@@ -2,23 +2,12 @@ package idtools // import "github.com/docker/docker/pkg/idtools"
 
 import (
 	"os"
-
-	"github.com/docker/docker/pkg/system"
-)
-
-const (
-	SeTakeOwnershipPrivilege = "SeTakeOwnershipPrivilege"
-)
-
-const (
-	ContainerAdministratorSidString = "S-1-5-93-2-1"
-	ContainerUserSidString          = "S-1-5-93-2-2"
```

Comment:
I think these may be used by BuildKit; we should probably provide temporary aliases for them as well.

Reply:
They are here but moved to a different file with Deprecation marker. They can't be removed anyway since the vendored buildkit uses them.